taskcluster/ci/clean-corpus/kind.yml

```diff
@@ -16,19 +17,33 @@ kind-dependencies:
 
 tasks:
     "{provider}-{dataset_sanitized}-{src_locale}-{trg_locale}":
-        description: Clean {provider} {dataset_sanitized} dataset {src_locale}-{trg_locale}
+        description: Clean {provider} {dataset_sanitized} dataset {src_locale}-{trg_locale} use OpusCleaner {use_opuscleaner}
         attributes:
             cleaning-type: clean-corpus
             dataset-category: train
             stage: clean-corpus
             cache:
                 type: clean-corpus
+                from-parameters:
+                    use_opuscleaner: training_config.experiment.use-opuscleaner
                 resources:
                     - pipeline/clean/clean-corpus.sh
                     - pipeline/clean/tools/deescape-special-chars.perl
                     - pipeline/clean/tools/remove-non-printing-char.perl
                     - pipeline/clean/tools/clean_parallel.py
                     - pipeline/clean/tools/langid_fasttext.py
+                    - pipeline/clean/opuscleaner/generate_filters.py
+                    - pipeline/clean/opuscleaner/clean-corpus.sh
+                    - pipeline/clean/opuscleaner/configs/remove_frequent_patterns.txt
+                    - pipeline/clean/opuscleaner/configs/default.filters.json
+                    - pipeline/clean/opuscleaner/configs/en-ru/mtdata_ELRC-3075-wikipedia_health-v1.en-ru.filters.json
```

Comment:
Question: What's the strategy here? Are we adding all of the filters for all languages and datasets here?